pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -553,7 +489,11 @@ func (ic *ifConfigurator) addPostInterfaceCreateHook(containerID, endpointName s
 		})
 
 		if pollErr != nil {
-			klog.ErrorS(err, "Failed to execute postInterfaceCreateHook", "interface name", ifaceName)
+			if err != nil {
+				klog.ErrorS(err, "Failed to execute postInterfaceCreateHook", "interface name", ifaceName)
+			} else {
+				klog.ErrorS(pollErr, "Failed to wait for host interface creation in 1min", "interface name", ifaceName)
```

Comment:
Not newly introduced, but let's fix the log format issue in this patch: structured logging key shouldn't contain space